lib/bracket-matcher.coffee

```diff
@@ -29,6 +29,25 @@ class BracketMatcher
     else
       @pairedCharacters = @defaultPairs
 
+  excludePairs: (excludePairs) ->
```

Comment:
Yes, I noticed that as well. It is now even more similar after the refactoring I did for performance. bracket-matcher.coffee was previously parsing and modifying the config options every time an edit operation was being performed. Ouch! I've changed it so that an update to config routine is called by the constructor; I then added callbacks to each of the config options so that the update config routine is called anytime the options change.

I ended up going back to bracket-matcher-view.coffee and putting in very similar behaviors. So yes, it should be refactored to have shared code, and only separate concerns where necessary.